src/seedsigner/views/seed_views.py

```diff
@@ -193,20 +194,44 @@ def run(self):
         
         elif button_data[selected_menu_num] == self.TYPE_12WORD:
             self.controller.storage.init_pending_mnemonic(num_words=12)
-            return Destination(SeedMnemonicEntryView)
+            if wordlist_language_code=="en": 
```

Comment:
Always use constants instead of literal strings.

Reply:
Also note typical style usage around `==`: space before and after in conditionals.